### DISCUSSION TITLE
How to use tag rules to not read a certain tag in a JSON file

### DISCUSSION THREAD

**Discussion by Nuo-Mituan** [Difegue/LANraragi#1094] [Q&A] [answered]
**LRR Version and OS**
Docker 0.9.0

**Bug Details**
LANraragi Documentation
tag | -tag : removes the tag
I have added all the permutations and combinations of this rule, but I cannot delete unnecessary tags after reading the file

![c93705b5-bab6-43ea-9019-e9c8d98f9e59](https://github.com/user-attachments/assets/12f0054c-b76d-4d27-8ef9-21265960d7ae)
![f78a14d5-53bd-46cb-ac02-bddb2e90fd82](https://github.com/user-attachments/assets/d17da237-073f-4313-b886-77a978dba68c)

**Comment by IceBreeze** [comment, marked as answer]
`tag | -tag` means use `tag` or `-tag`.

`tag` is there for compatibility with the previous blacklist only system.

Try `-wings` in your case because the "Tag" label is for items without namespace.

**Comment by Nuo-Mituan** [comment]
Thank you. The issue has been resolved
